Sources/HaCWebsiteLib/Models/NewWorkshop.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import Yaml
+
+/// Represents a workshop
+// TODO: Rename this struct to `Workshop` when it's ready for use
+public struct NewWorkshop {
+  let title: String
+
+  /// People that have directly contributed content to this workshop
+  let contributors: [String]
+
+  /// Non-contributors we would like to thank for helping to inspire, develop or facilitate this workshop
+  let thanks: [String]
+
+  /// Links that attendees may which to read after the workhsop
+  let furtherReadingLinks: [(String, URL)]
+
+  /// A link to a video recording of this workshop
+  let recordingLink: URL?
```

Comment:
Minor but shouldn't we have any optional fields underneath all the mandatory ones?

Reply:
Don't think so necessarily. The default member-wise initialiser still makes us explicitly say `nil` for these so it doesn't make much difference. They're in this order because this is roughly the order they're in in the workshops repos.